opm/parser/eclipse/EclipseState/Aquifer/NumericalAquifer/NumericalAquiferCell.hpp

```diff
@@ -28,19 +29,20 @@ namespace Opm {
     class FieldPropsManager;
 
     struct NumericalAquiferCell {
-        NumericalAquiferCell(const DeckRecord&, const EclipseGrid&, const FieldPropsManager&);
+        NumericalAquiferCell(const std::size_t record_id_, const DeckRecord&, const EclipseGrid&, const FieldPropsManager&);
         NumericalAquiferCell() = default;
-        size_t aquifer_id;
-        size_t I, J, K;
-        double area;
-        double length;
-        double porosity;
-        double permeability;
-        double depth;
-        std::optional<double> init_pressure;
-        int pvttable;
-        int sattable;
-        size_t global_index;
+        std::size_t aquifer_id{};
```

Comment:
Does this `{}` really give us anything?

Reply:
Yes.  It means we reliably initialise to zero rather than get whatever bits happened to be in the memory location before.

Reply:
Well - I disagree; the reliably initialized to zero instance is not usable. The only way to get a valid usable object is through the constructor.

Reply:
> The only way to get a valid usable object is through the constructor.

True, but this is needed to support the type's default constructor.  The defaulted default constructor doesn't initialise built-in arithmetic types.